styles.css

```diff
@@ -343,25 +389,27 @@
   border-radius: 15px;
 }
 
-.pd-1 {
+/* || Padding  */
+.pd-4 {
   padding: 4px;
 }
 
-.pd-2 {
+.pd-8 {
   padding: 8px;
 }
 
-.pd-3 {
+.pd-16 {
   padding: 16px;
 }
 
-.pd-4 {
+.pd-24 {
   padding: 24px;
```

Comment:
can use rems instead of pixels

Reply:
It's my style guide basically for consistent units across all screen sizes.